runtime/src/main/resources/application.yml

```diff
@@ -80,3 +80,11 @@ generator:
 controllers:
   integration:
     enabled: true
+
+client:
+  state:
+    authenticationAlgorithm: HmacSHA1
```

Comment:
I still would like these removed from the committed codebase due to the potential problems I mentioned above.

We could prevent startup if these values aren't set perhaps so we know they have to be set in prod deployments?

Keys should be regenerated, short lived keys are great, but you're right we need to think about how we do that. I'd recommend we do a rolling deployment rather than updating in place, but we also need to figure out how we accept old encrypted cookies and re-encrypt on next request if we do that. But this can be future PR work IMO.